.github/workflows/test-self-hosted.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         
-      - uses: prefix-dev/setup-pixi@v0.4.3
+      - uses: prefix-dev/setup-pixi@v0.5.0
         with:
           cache: true
```

Comment:
you need `environments: test` here. Otherwise it will only cache the `default` environment.